### PR TITLE
Faster icon

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] To render a small storysheet: This pr: avg 80ms, Master avg: 148ms

### CODE REVIEW

File: shared/common-adapters/icon.native.js [hunk 1]
```diff
@@ -2,159 +2,192 @@
...
+      ref={ref}
+      fontSize={fontSize}
+      onPress={p.onClick}
+    >
+      {p.children}
+    </Kb.NativeText>
+  )
+}
+
+Text = React.forwardRef(Text)
```
Comment (chrisnojima) [inline shared/common-adapters/icon.native.js]: this matches the style internal to RN

**Comment by chrisnojima** [comment]
@adamjspooner wanna sanity check what this looks like on mobile. it should be the same logic as before

**Comment by chrisnojima** [comment]
(no rush, for after the release)

**Comment by adamjspooner** [comment]
The icon sizes on this branch are significantly smaller than master. Otherwise, everything looks good to me.

icons

**Comment by chrisnojima** [comment]
@adamjspooner fixed i think

**Comment by adamjspooner** [comment]
Looks good to me!

**Pull request merged**